dev/sparktestsupport/modules.py

```diff
@@ -469,7 +469,7 @@ def __hash__(self):
     name="yarn",
     dependencies=[],
     source_file_regexes=[
-        "yarn/",
+        "resource-managers/yarn/",
```

Comment:
@vanzin If I understand correctly, the sbt test goals can stay the same? 
I can see them defined the same way for other modules such as `sketch` (as `sketch/test`), but it is nested under `common/`.